app/src/main/java/com/codesoom/assignment/ProductNotFoundException.java

```diff
@@ -1,5 +1,8 @@
 package com.codesoom.assignment;
 
+/**
+ * 상품을 찾을 수 없다는 예외.
+ */
```

Comment:
이런 경우에는 다음과 같이 작성하는 것을 고려할 수 있을 거에요.
```suggestion
/**
 * 찾을 수 없는 상품 예외.
 */
```
```suggestion
/**
 * 상품을 찾을 수 없는 경우에 던집니다.
 */
```

Reply:
자바 라이브러리의 Exception 클래스들을 찾아보니 "Thrown when~", "Thrown to indicate that~" 으로 시작하는 경우가 많이 보이네요. 무슨 예외인지 보다 어떤 상황에서 이 예외가 던져지는지에 초점을 맞춰야겠다는 생각이 들었습니다. 2번째로 제안해 주신 문장으로 수정하겠습니다!